public/js/all.js

```diff
@@ -1,7 +1,3 @@
-function asset(uri) {
```

Comment:
This is the only one I'm concerned with. I think it's only used in loadDynamicTooltip.ts, but after removing it and rebuilding everything, the dynamic tooltip still loads, so maybe it's using the PHP version of the function?

Reply:
This should be okay. It's using `asset()` from _helpers.ts_:

```
// loadDynamicTooltip.ts
import { asset, fetcher } from '@/utils';

// resources/js/utils/index.ts
export * from './helpers';

// helpers.ts
export const asset = (uri: string) => { ... };
```